filecoin-proofs/tests/paramfetch/prompts_to_fetch.rs

```diff
@@ -146,3 +155,50 @@ fn fetches_vk_even_if_sector_size_does_not_match() -> Result<(), FailureError> {
 
     Ok(())
 }
+
+#[test]
+fn invalid_json_path_produces_error() -> Result<(), FailureError> {
+    let mut session = ParamFetchSessionBuilder::new(Some(PathBuf::from("/invalid/path")))
+        .with_session_timeout_ms(1000)
+        .build();
+
+    session.exp_string("fatal error: json file '/invalid/path' does not exist")?;
```

Comment:
JSON is an acronym. Please update to read: "fatal error: JSON file '/invalid/path' does not exist.